maxim/logger/openai/utils.py

```
@@ -1,5 +1,6 @@
 import time
 from typing import Any, Dict, Iterable, List, Optional, Union
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)